pkgs/development/tools/datree/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "datree";
+  # test
+  version = "0.13.4";
+
+  src = fetchFromGitHub {
+    owner = "datreeio";
+    repo = "datree";
+    rev = "${version}";
+    sha256 = "0q399ydz6k3p88dlyk5qzki76wiiw3zvkfh40vh9y9wzala8dhhp";
+  };
+
+  vendorSha256 = "1l7k1ss0iqlkjldi0jqlpfrlzj1zkmbzi98m5p3d12rr84i7in2k";
+
+  ldflags = let t = "sigs.k8s.io/kustomize/api/provenance"; in
+    [
+      "-s"
+      "-X github.com/datreeio/datree/cmd.CliVersion=${version}"
+    ];
+
+  doCheck = false;
```

Comment:
Why did you disable them?

Reply:
No specific reason, I copied it from another package. I enabled them now.